### PR TITLE
patch eslint plugin import

### PR DESCRIPTION
**Pull request by strawberry-choco**
Signed-off-by: Kangrui Ye <kangrui.ye@cloudflight.io>

### CODE REVIEW

File: packages/eslint-plugin-angular/package.json [hunk 1]
```diff
@@ -26,13 +26,14 @@
...
+        "@angular-eslint/template-parser": "13.5.0"
+    },
     "peerDependencies": {
-        "@angular-eslint/eslint-plugin": ">=13.0.0",
-        "@angular-eslint/eslint-plugin-template": ">=13.0.0",
-        "@angular-eslint/template-parser": ">=13.0.0",
         "@cloudflight/eslint-plugin-typescript": ">=0.1.0",
-        "@typescript-eslint/parser": ">=5.0.0 <6.0.0",
-        "eslint": ">=8.0.0 <9.0.0"
+        "eslint": "8.22.0"
```
Comment (cgrabmann) [inline packages/eslint-plugin-angular/package.json:36]: why would you explicitly restrict it to one exact version?

Comment (strawberry-choco) [reply packages/eslint-plugin-angular/package.json:36]: To prevent any potential breakage.

Comment (strawberry-choco) [reply packages/eslint-plugin-angular/package.json:36]: Especially between typescript-eslint and eslin itself.

Comment (cgrabmann) [reply packages/eslint-plugin-angular/package.json:36]: yes but that forces a project to use exactly this version, which might be an issue for them.

In my opinion `peerDependencies` should always be as lenient as possible and avoid depending on a fixed version, then that could be a normal dependency from the start.

Comment (strawberry-choco) [reply packages/eslint-plugin-angular/package.json]: Fixed.

Comment (cgrabmann) [reply packages/eslint-plugin-angular/package.json]: eslint-plugin-typescript still has the exact version

Comment (strawberry-choco) [reply packages/eslint-plugin-angular/package.json]: Fixed.

**Review by cgrabmann — APPROVED** [review]

**Pull request merged**